### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by deejay1**
Fixes some minor typing errors in README.md files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,4 +59,4 @@ ObjectMapper mapper = JsonMapper.builder()
     .build();
 ```
 
-after which datatype read/write support is available for all normal Jackson operations,
+after which datatype rewad/write support is available for all normal Jackson operations,
```
Comment (cowtowncoder) [inline README.md:62]: Maybe not this change. :)

**Comment by cowtowncoder** [comment]
Thanks! Just one change needed, can merge it then.

File: README.md (hunk 1 above)
Comment (deejay1) [reply README.md]: Good catch. Sorry I've missed it reviewing the diff :/

**Comment by cowtowncoder** [comment]
Np, thanks!

**Pull request merged**